cmd/config.go

```diff
@@ -157,6 +174,33 @@ func RunPhases(phase string, v reflect.Value) {
 	}
 }
 
+// ReloadServices iterates through key/values calling reload on applicable services.
+func ReloadServices(v reflect.Value) {
+	for i := 0; i < v.NumField(); i++ {
+		if reflect.Value.IsZero(v.Field(i)) {
+			continue
```

Comment:
Add a comment here please, 
Suggestion:
"If the service is not part of the backends struct, do nothing"